region/maridia/inner-yellow/Thread The Needle Room.json

```diff
@@ -216,6 +534,30 @@
       ],
       "note": "Dodge the Puyos and kill the Choots. Expects two to three Puyo hits."
     },
+    {
+      "link": [1, 2],
+      "name": "Come In Getting Blue Speed, Leave With Temporary Blue",
```

Comment:
I'd like to see this as a notable - it is actually threading the needle.
Feels like extreme to me for both.
You can also unmorph a few times to do a very slow springball bounce across, but you take a lot of damage that way.